python/dgl/graphbolt/feature_store.py

```diff
@@ -52,6 +52,16 @@ def update(self, value: torch.Tensor, ids: torch.Tensor = None):
         """
         raise NotImplementedError
 
+    def metadata(self):
```

Comment:
existing `.size()` is one of the metadata? Or a special metadata that lies standalone?

Reply:
Size is actually inferred from the tensor.